src/Organic/GraphQL.php

```diff
@@ -144,7 +144,7 @@ public function register_organic_types() {
                     $adsConfig = $organic->getAdsConfig();
                     $testEnabled = $organic->useSplitTest();
                     return [
-                        'organicEnabled' => $organic->isEnabledAndConfigured(),
+                        'organicEnabled' => $organic->isConfigured(),
```

Comment:
Right now, a lot of Recurrent sites do have Organic WP plugin enabled and configured in WP, but the "integration" checkbox is disabled

That's because Organic Web has a hard dependency on Organic WP plugin GQL API and they can't just disable/delete the whole plugin

And by passing the `$organic->isConfigured()` as `organicEnabled` value will lead to injecting and setting up Organic SDK on those sites (even if Ads and Affiliate are disabled the Core is always enabled in the SDK)
https://github.com/orgnc/organic-content/blob/ae0c8844ae433483ac26300a8f8a2b8489490a19/frontend/packages/core/components/common/OrganicSDK/DocumentHeadOrganicTags.tsx#L184

Reply:
> That's because Organic Web has a hard dependency on Organic WP plugin GQL API

Ah, ok, I see. I wonder if we can modify the language or add plugin field descriptions to make this all clearer? As a client looking at the settings page, I would be extremely confused. What does it mean for the integration to be enabled--isn't the plugin already active? If it's enabled but Ads, Affiliate, and Campaigns are not, then what is the plugin doing? Etc.

Tagging @cjkie here too.